content/blog/_posts/inclusion-good-for-business.md

```diff
@@ -0,0 +1,47 @@
+
+AI still has some kinks to work out, though, such as [algorithmic bias](https://www.youtube.com/watch?v=UG_X_7g63rY), which exists, again, as a result of the technologists who created it. When early developers were building and testing these tools, they used themselves—who were neither an ethnically nor racially diverse set of subjects—as the baseline. As these algorithms have been put into practice for facial pattern recognition and surveillance, they’re not wholly effective—and, as a source of racial profiling and misidentification—are often dangerous. So, it’s important to thoroughly investigate the tools you may use to ensure they’re using AI responsibly.
+
+
+## Turn inclusive language into an inclusive culture
+
+Putting more care and attention into inclusive language when hiring is bit of a self-fulfilling prophecy—you’re actively building a more diverse and inclusive workforce. From, there, the onus is on companies, past the point of making a diverse hire, to also actively include those new hires in organizational activities and decision-making to build a truly inclusive culture.
+
+More prospective employees are making that a dealbreaker, too. In Lenovo and Intel’s 2020 brief, _[Diversity & Inclusion in the Global Workplace](https://news.lenovo.com/wp-content/uploads/2020/07/LENOVO_INTEL_D_I_Research-Brief_7.2020.pdf)_, over half of its global survey respondents said that a company’s diversity and inclusion policies were “extremely” or “very” important when deciding where to apply and whether to accept an offer.
+
+AI, machine learning, and advanced analytics are also helping companies keep a better eye on how their employees feel about their workplace and their jobs through employee surveys and well-being assessments that gauge satisfaction levels, and algorithmic matching to advancement opportunities.
+
+That said, there are still gaps to fill between action and perception. A recent multi-year [PwC study](https://www.pwc.com/gx/en/services/people-organisation/global-diversity-and-inclusion-survey.html) found that while 63 percent of business leaders believe they’re doing a good job communicating about employee and leadership diversity, only 42 percent of employees agree with them—and 33 percent feel that diversity is a barrier to their progression within the organization.
+
+At the end of the day, implementing DEI creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.
```

Comment:
At the end of the day, implementing DEI and robust Inclusive Naming policies creates equitable opportunities for everyone to succeed, and that’s just good business, with both social and financial returns on investment. _[The Wall Street Journal](https://www.wsj.com/articles/the-business-case-for-more-diversity-11572091200)_’s research of DEI across the S&P 500 discovered that the 20 most diverse companies among those it surveyed have better operating results on average than the lowest-scoring firms, and their shares typically outperform those of the least-diverse firms, proving that inclusion is not only the right thing to do but is also the business imperative of our time.